docker-compose.yml

```diff
@@ -1,25 +1,25 @@
 version: '3'
 services:
   app:
-    image: bpanel/bpanel
+    ## Uncomment to build
```

Comment:
Uncomment to build? It already is not commented out

Reply:
d'oh. Was testing the fix for npm uninstalling my dependencies. will put it back to build  from remote image.